test-framework/junit5/src/main/java/io/quarkus/test/junit/DisabledOnIntegrationTestCondition.java

```diff
@@ -1,10 +1,14 @@
 package io.quarkus.test.junit;
 
+import static io.quarkus.test.junit.ArtifactTypeUtil.*;
+import static io.quarkus.test.junit.DisabledOnIntegrationTest.ArtifactType.*;
+import static io.quarkus.test.junit.IntegrationTestUtil.*;
```

Comment:
I was under the impression that we don't do star imports? Other classes are affected as well.

Reply:
Good point, fixed!